.github/workflows/test.yml

```diff
@@ -38,11 +38,11 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
       - name: CMake Configure
-        run: cmake -B out -DCMAKE_BUILD_TYPE=Release ${{ runner.os == 'macOS' && '-DBASE64_WITH_AVX2=OFF' || '' }}
+        run: cmake -B out -Werror=dev -DBASE64_BUILD_TESTS=ON ${{ runner.os != 'Windows' && '-DCMAKE_BUILD_TYPE=Release' || '' }} ${{ runner.os == 'macOS' && '-DBASE64_WITH_AVX2=OFF' || '' }}
```

Comment:
This command should be split as follows for readability:

```yaml
run: >
  cmake
  -B out
  -Werror=dev
  -DBASE64_BUILD_TESTS=ON
  ${{runner.os != 'Windows' && '-DCMAKE_BUILD_TYPE=Release' || ''}}
  ${{runner.os == 'macOS' && '-DBASE64_WITH_AVX2=OFF' || ''}}
```

Reply:
done